### PR TITLE
DOCSP-36989 Count

### PR DESCRIPTION
**Pull request by jordan-smith721**
# Pull Request Info

[PR Reviewing Guidelines](https://github.com/mongodb/docs-node/blob/master/REVIEWING.md)

JIRA - https://jira.mongodb.org/browse/DOCSP-36989
Staging - https://preview-mongodbjordansmith721.gatsbyjs.io/pymongo/DOCSP-36989-count/fundamentals/crud/count/

## Self-Review Checklist

- [x] Is this free of any warnings or errors in the RST?
- [x] Did you run a spell-check?
- [x] Did you run a grammar-check?
- [x] Are all the links working?
- [x] Are the [facets and meta keywords](https://wiki.corp.mongodb.com/display/DE/Docs+Taxonomy) accurate?

### CODE REVIEW

File: source/fundamentals/crud/count.txt [hunk 1]
```diff
@@ -1,16 +1,40 @@
...
+   :name: genre
+   :values: reference
+
+To see how many documents are in a collection, use the
+``~pymongo.collection.Collection.count_documents`` method.
+
+Count All Documents
+-------------------
+
+To return a count off all documents in the collection, pass an empty object to
```
Comment (mongoKart) [inline source/fundamentals/crud/count.txt:25]:
```suggestion
To return a count of all documents in the collection, pass an empty object to
```

File: source/fundamentals/crud/count.txt [hunk 2]
```diff
@@ -1,16 +1,40 @@
...
 .. code-block:: python
 
    >> posts.count_documents({})
    3
 
-or just of those documents that match a specific query:
+Count Specific Documents
+------------------------
+
+To return a count of documents that match a specific search criteria, specify a
```
Comment (mongoKart) [inline source/fundamentals/crud/count.txt:36]:
```suggestion
To return a count of documents that match specific search criteria, specify a
```

File: source/fundamentals/crud/count.txt [hunk 3]
```diff
@@ -1,16 +1,40 @@
...
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+.. facet::
+   :name: genre
+   :values: reference
+
+To see how many documents are in a collection, use the
+``~pymongo.collection.Collection.count_documents`` method.
```
Comment (mongoKart) [inline source/fundamentals/crud/count.txt:20]:
```suggestion
``~pymongo.collection.Collection.count_documents()`` method.
```

File: source/fundamentals/crud/count.txt [hunk 4]
```diff
@@ -1,16 +1,40 @@
...
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+.. facet::
+   :name: genre
+   :values: reference
+
```
Comment (mongoKart) [inline source/fundamentals/crud/count.txt:18]:
```suggestion

.. meta::
   :keywords: number
   
```

or whatever you think makes sense here

**Review by mongoKart — APPROVED** [review]
lgtm + small changes

File: source/fundamentals/crud/count.txt (hunk 4 above)
Comment (jordan-smith721) [reply source/fundamentals/crud/count.txt:18]: Thanks, I couldn't think of anything for this page so that works

**Pull request merged**